src/components/dateTimePicker/index.tsx

```diff
@@ -237,7 +243,7 @@ class DateTimePicker extends Component {
           iconStyle={{tintColor: Colors.grey10}}
           onPress={this.toggleExpandableOverlay}
         />
-        <Button link iconSource={Assets.icons.check} useCustomTheme={useCustomTheme} onPress={this.onDonePressed}/>
+        <Button link iconSource={Assets.icons.check} useCustomTheme onPress={this.onDonePressed}/>
```

Comment:
Are you sure we want to pass `useCustomTheme` as true? Don't we want to ignore the ts error?

Reply:
Im assuming it will always need to apply useCustomTheme 
But I see your point, added it back and ignored